solr/modules/ltr/src/test/org/apache/solr/ltr/TestLTRQParserPlugin.java

```diff
@@ -78,6 +79,60 @@ public void ltrModelIsEmptyTest() throws Exception {
     assertTrue(res.contains("the model 0 is empty"));
   }
 
+  @Test
+  public void ltrRepeatedQueriesCachedTest() throws Exception {
+    final SolrQuery query = new SolrQuery();
+    query.setQuery("bloomberg baz buzz");
+    query.add("q", "bloomberg baz buzz");
+    query.add("qf", "title description");
+    query.add("defType", "edismax");
+    query.add("fl", "*, score");
+    query.add("rows", "4");
+    query.add("fv", "true");
+    query.add("rq", "{!ltr model=6029760550880411648 reRankDocs=3}");
```

Comment:
wondering how practical it would be to have test coverage for the interleaving case too.